delivery/repositories.py

```diff
@@ -0,0 +1,47 @@
+from abc import ABC, abstractmethod
+from delivery.error_handling import WrongIdError
+
+
+class AbstractRestaurantRepository(ABC):
+    @abstractmethod
+    def create(self, content):
+        pass
+
+    @abstractmethod
+    def update(self, content, restaurant_id):
+        pass
+
+    @abstractmethod
+    def get_all(self):
+        pass
+
+    @abstractmethod
+    def get_by_id(self, id):
+        pass
+
+
+class MemoryRestaurantRepository(AbstractRestaurantRepository):
+    restaurants = []
+    # def __init__(self, restaurants=None):
+    #     self.restaurants = restaurants or []
+
+    def create(self, content):
+        content.id = len(self.restaurants)
+        self.restaurants.append(content)
+
+    def update(self, content_dictionary, restaurant_id):
```

Comment:
Yes, I use it for post request and others. But in case with put I haven't got all parameters for my object `Restaurant`. So I give a dict but update all parametrs from dict in object that saved in  list of `Restaurant` s

Reply:
fixed